src/sentry/tasks/auto_resolve_issues.py

```diff
@@ -60,7 +60,10 @@ def schedule_auto_resolution():
         if int(options.get("sentry:_last_auto_resolve", 0)) > cutoff:
             continue
 
-        auto_resolve_project_issues.delay(project_id=project_id, expires=ONE_HOUR)
+        auto_resolve_project_issues.apply_async(
+            kwargs={"project_id": project_id, "expires": ONE_HOUR},
```

Comment:
Do you want this task to expire in one hour? Or does the task take `expires` as a parameter? Looking at the task definition it doesn't seem to use `expires`, but in its current position it won't set an expiration on the task either.

Reply:
yes, i was confused. just updated.